std_msgs/Header header
float64 latitude 
float64 longitude 
float64 altitude
int64 gps_quality
float64 utm_easting
float64 utm_northing
float64 zone_number
string zone_letter
